src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+// const TodoForm = props => <textarea placeholder="Enter stuff here"></textarea>
+
+// const AddToDo = props => <button>Add stuff</button>
+
+// const TodoForm = () => {
+//     return (
+//         <div>
+//             <form onSubmit={props.addToDo} placeholder="Enter stuff here"></form>
+//             <button>Add Todo</button>
+//             <button>Clear Completed</button>
+//         </div>
+//     )
+// }
+
+// const TodoForm = props => {
```

Comment:
I know leaving comments in isn't the best thing in a professional setting but I think it's really good when you're learning so you can always go back and reference your code.